munit/js/src/main/scala/munit/internal/PlatformCompat.scala

```diff
@@ -75,8 +75,4 @@ object PlatformCompat {
   private var myClassLoader: ClassLoader = _
   def setThisClassLoader(loader: ClassLoader): Unit = myClassLoader = loader
   def getThisClassLoader: ClassLoader = myClassLoader
-
-  type InvocationTargetException = munit.internal.InvocationTargetException
-  type UndeclaredThrowableException =
-    munit.internal.UndeclaredThrowableException
```

Comment:
I think we can also remove these type aliases from the corresponding JVM/Native files.

Reply:
Nice catch